### PR TITLE
rem 20150618

### PR DESCRIPTION
**Pull request by canmor-lam**
*rem* is a Command-line tool to access OSX Reminders.app database. Home page: https://github.com/kykim/rem

### CODE REVIEW

File: Library/Formula/rem.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                /usr/local/Library/Contributions/example-formula.rb
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (dunn) [inline Library/Formula/rem.rb]: Remove all the comments, please!

File: Library/Formula/rem.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  version "20150618"
```
Comment (dunn) [inline Library/Formula/rem.rb]: Homebrew should be able to detect the version from the url string without needing `version` made explicit.  Run `brew audit --strict` to doublecheck.

File: Library/Formula/rem.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  # depends_on "cmake" => :build
+
+  def install
+    # ENV.deparallelize  # if your formula fails when building in parallel
+
+    # Remove unrecognized options if warned by configure
+    system "xcodebuild"
+    system "mkdir", "#{prefix}/bin"
```
Comment (dunn) [inline Library/Formula/rem.rb]: This line and the next can be simplified into a single line: `bin.install "build/Release/rem"`

File: Library/Formula/rem.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! It's enough to just replace
+    # "false" with the main program this formula installs, but it'd be nice if you
+    # were more thorough. Run the test with `brew test rem`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    system "#{bin}/rem", "version"
```
Comment (dunn) [inline Library/Formula/rem.rb]: A more substantial test would be great; something that checks general functionality.

**Comment by canmor-lam** [comment]
Hi, I just updated the commit. I'm sorry for boring you, thanks for your patience and suggestions.

File: Library/Formula/rem.rb (hunk 4 above)
Comment (canmor-lam) [inline Library/Formula/rem.rb]: You are right, but in this case, any general functionality will trigger OS X to ask user to authorize accessing Reminders.

File: Library/Formula/rem.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  def install
+    xcodebuild
```
Comment (DomT4) [inline Library/Formula/rem.rb]: There are no necessary flags to pass to this at all?

Comment (canmor-lam) [inline Library/Formula/rem.rb]: Yep, [origin project](https://github.com/kykim/rem) says:

```
git clone https://github.com/kykim/rem.git
cd rem
xcodebuild
cp build/Release/rem /usr/local/bin
```

Also, I did test that.

Comment (DomT4) [inline Library/Formula/rem.rb]: Cool. No, no worries, sometimes `xcodebuild` by itself is fine but just wanted to check.

Can you add `depends_on :xcode => :build` one empty line below `sha256` and then we should be good to go on this.

Comment (canmor-lam) [inline Library/Formula/rem.rb]: Thank you for your kindly help, I'd add `depends_on :xcode => :build` and rebased up-to-date.

Thought that xcode must be already installed during home-brew install.  ;)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/37621ec4e1182b52265701fe51a5f67704e1b198. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**